docs/DeveloperGuide.md

```diff
@@ -190,6 +190,57 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+
+<u>**Use case: UC07 - Add an interview**</u>
+
+**MSS**
+
+1. User requests to <u>list all positions (UC05)</u>.
+2. User requests to <u>list all candidates (UC03)</u>.
+3. User requests to add an interview with details for the interview.
+3. HR Manager adds interview.   
```

Comment:
Shouldn't there be only one 3. ?

Reply:
Oops, thanks for catching it, just changed it